src/code/z_actor.c

```diff
@@ -2902,10 +2902,10 @@ void Actor_DrawAll(PlayState* play, ActorContext* actorCtx) {
     gSPDisplayList(sp58, &ref2[1]);
     POLY_XLU_DISP = &ref2[1];
 
-    if (play->actorCtx.unk3 != 0) {
+    if (play->actorCtx.lensActive) {
         Math_StepToC(&play->actorCtx.unk4, 100, 20);
```

Comment:
There's a few lens things I can name here if you'd like:
`actorCtx.unk3` -> `actorCtx.lensToggleTimer`? (or maybe just `actorCtx.lensTimer`)
`actorCtx.unkB` -> `actorCtx.lensActorsDrawn`
`func_800B9EF4` -> `Actor_DrawLensActors`
`func_800B9E84` -> `Actor_DrawLensOverlay`
`D_801DE890` -> `gLensOfTruthMaskTex`

Reply:
Yeah, provided we're clear on what's handled differently from OoT here, I think that's a good idea. But maybe not for this PR?

Reply:
afaik, the actual overall purpose of all these functions are the same, it just has slightly different implementation:

- I confirmed `gLensOfTruthMaskTex` was the same as OoT with Texture64
-  `Actor_DrawLensOverlay` is definitely drawing the purple circle, but goes through the fbdemo system instead of drawing it directly
- `Actor_DrawLensActors` is definitely drawing lens actors, but the implementation is different
- `actorCtx.lensActorsDrawn` seems like an accurate name
- You're original comment, `actorCtx.unk4` -> `actorCtx.lensMaskRadius` is actually a bad name because while this variable does control the size of the lens circle drawn on screen, it's proportional to `1 - unk4`, i.e. a value close to 0 will draw the circle at it's widest, and a value of 100 will draw the circle at a smaller radius. So it does relate to the size of the circle, but inversely (and not multiplicatively inverse, but through the additive inverse if there's a word for that)

Reply:
Maybe something like "fully on proportion", but actually good.